test/deployment/resources/resource_request_test.go

```diff
@@ -35,6 +35,23 @@ var _ = ginkgo.Describe("ResourceRequestsTests", func() {
 		deployBroker(brokerDeployer)
 		pod := getPod(ctx1)
 		actualMemRequest := pod.Spec.Containers[0].Resources.Requests.Memory()
-		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()))
+		gomega.Expect(expectedMemRequest).To(gomega.Equal(actualMemRequest.String()), "Expected Memory limit: %s, real: %s", expectedMemLimit, actualMemLimit.String())
+	})
+
+	ginkgo.It("Memory Request check", func() {
```

Comment:
@mkrutov this test looks like a improvement of the test on L32, So, should the one starting on L32 be removed?

Reply:
Discussed in google chat